flask_jwt_router/_jwt_routes.py

```diff
@@ -99,7 +99,7 @@ def register():
         def login():
             # I'm authorized & updating my token!
             return jsonify({
-                "token": jwt_routes.update_entity(entity_id=1)
```

Comment:
If you can instal `Sphinx` &  run the following:
```
make html
```
I will update the Pipfile to add Sphinx as a dev dep in my Pr either today or tomorrow, thank you.

Reply:
Okay I'll try and give it a go!